qiita_db/test/test_artifact.py

```diff
@@ -42,14 +42,31 @@ def test_iter_public(self):
         exp = []
         self.assertEqual(obs, exp)
 
-    def test_types(self):
+    def test_create_type(self):
         obs = qdb.artifact.Artifact.types()
         exp = [['BIOM', 'BIOM table'],
                ['Demultiplexed', 'Demultiplexed and QC sequeneces'],
                ['FASTA', None], ['FASTA_Sanger', None], ['FASTQ', None],
                ['SFF', None], ['per_sample_FASTQ', None]]
         self.assertItemsEqual(obs, exp)
 
+        qdb.artifact.Artifact.create_type(
+            "NewType", "NewTypeDesc", False, False,
+            [("log", False), ("raw_forward_seqs", True)])
+
+        obs = qdb.artifact.Artifact.types()
+        exp = [['BIOM', 'BIOM table'],
+               ['Demultiplexed', 'Demultiplexed and QC sequeneces'],
```

Comment:
Good catch, opened https://github.com/biocore/qiita/issues/1961 cause this is a larger issue.